src/config.h

```diff
@@ -91,6 +91,15 @@
 #define HAVE_ACCEPT4 1
 #endif
 
+/* Detect for pipe2() */
+#if defined(__linux__) || \
+    defined(__FreeBSD__) || \
+    (defined(__OpenBSD__) && OpenBSD >= 201505) || \
+    (defined(__DragonFly__) && __DragonFly_version >= 400106) || \
+    (defined(__NetBSD_Version__) && __NetBSD_Version__ >= 600000000)
```

Comment:
but on second though, should this way feel more explicit? @skaslev 

```suggestion
#if defined(__linux__) || \
    defined(__FreeBSD__) || \
    (defined(__OpenBSD__) && OpenBSD >= 201505) || \
    (defined(__DragonFly__) && defined(__DragonFly_version) && __DragonFly_version >= 400106) || \
    (defined(__NetBSD__) && defined(__NetBSD_Version__) && __NetBSD_Version__ >= 600000000)
```

Reply:
Based on examples as for [OpenBSD](https://github.com/openbsd/src/blob/42ad4d6e1ee006c92498feb6144bc05f362fa78d/lib/libcurses/tinfo/lib_baudrate.c#L58), [NetBSD](https://github.com/DragonFlyBSD/DragonFlyBSD/blob/d41d1a09426a4611e71e6565641f34d0ffc1cc4d/contrib/dhcpcd/src/ipv6.h#L106-L107)
```suggestion
#if defined(__linux__) || \
    defined(__FreeBSD__) || \
    (defined(__OpenBSD__) && OpenBSD >= 201505) || \
    (defined(__DragonFly_version) && __DragonFly_version >= 400106) || \
    (defined(__NetBSD_Version__) && __NetBSD_Version__ >= 600000000)
```
seems reasonable too.